contracts/uma/AnteLSPCollateralTest.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import "../interfaces/IERC20.sol";
+
+interface LongShortPair {
+    function longToken() external view returns (address);
+    function shortToken() external view returns (address);
+}
+
+/// @notice Ensure that issued tokens are less than or equal to LSP collateral * collateral per pair
+contract AnteLSPCollateralTest is AnteTest("Ensure that collateral x CPP is correct") {
+
+    LongShortPair private immutable contractLSP;
+    address private immutable addressLSP;
+    address private immutable addressToken;
+
+    address private immutable addressLong;
+    address private immutable addressShort;
+
+    IERC20 private immutable contractToken;
+    IERC20 private immutable contractLong;
+    IERC20 private immutable contractShort;
+
+    /// @param _addressLSP The address of the Long Short Pair
+    /// @param _addressToken The address of the token that is being bet on
+    constructor (address _addressLSP, address _addressToken) {
+        protocolName = "UMA";
+        testedContracts = [_addressLSP];
+
+        addressLSP = _addressLSP;
+        addressToken = _addressToken;
```

Comment:
The collateral token you can read from the LongShortPair, so it could be cleaner if you only need to add in `_addressLSP`